itou/www/job_seekers_views/views.py

```diff
@@ -50,6 +50,22 @@
 logger = logging.getLogger(__name__)
 
 
+def jobseeker_personal_infos_display_kwargs(jobseeker_profile):
+    other_infos_keys = [
+        "resourceless",
+        "unemployed_since",
+        "rqth_employee",
+        "oeth_employee",
+        "rsa_allocation_since",
+        "ass_allocation_since",
+        "aah_allocation_since",
+    ]
+    return {
+        "show_birth_place": not jobseeker_profile.birth_country or jobseeker_profile.birth_country.code == "100",
+        "other_infos": [key for key in other_infos_keys if getattr(jobseeker_profile, key, False)],
```

Comment:
Ces attributs existeront toujours.
```suggestion
        "other_infos": [key for key in other_infos_keys if getattr(jobseeker_profile, key)],
```